dGame/dComponents/PropertyEntranceComponent.h

```diff
@@ -59,10 +59,6 @@ class PropertyEntranceComponent final : public Component {
 	 */
 	[[nodiscard]] LWOMAPID GetMapID() const { return m_MapID; };
```

Comment:
```suggestion
	[[nodiscard]] LWOMAPID GetMapID() const noexcept { return m_MapID; };
```
the nitpickiest of nitpicks: a simple getter for a basic integer type can be noexcept

Reply:
but what if